tutorials/classiq_concepts/execute/advanced_execution/advanced_execution_user_guide.ipynb

```diff
@@ -0,0 +1,776 @@
+{
```

Comment:
<blockquote>execution of a quantum program with two varying parameters</blockquote><p>I believe that it should be something more like "execution of a quantum program with different varying parameters", since we can use <code>ExecutionSession</code> to vary more than two parameters at the same time</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/597/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>